src/applications/mhv-medications/components/MedicationsList/MedicationsListFilter.jsx

```diff
@@ -114,12 +120,23 @@ const MedicationsListFilter = props => {
             />
           ))}
         </VaRadio>
-        <VaButton
-          className="vads-u-width--full tablet:vads-u-width--auto filter-submit-btn vads-u-margin-top--3"
-          onClick={handleFilterSubmit}
-          text="Apply filter"
-          data-testid="filter-button"
-        />
+        <div>
+          <VaButton
+            className="vads-u-width--full tablet:vads-u-width--auto filter-submit-btn vads-u-margin-top--3"
+            onClick={handleFilterSubmit}
+            text="Apply filter"
+            data-testid="filter-button"
+          />
+        </div>
+        <div>
```

Comment:
could we remove the extra div's here? It adds an unnecessary empty element to the accessibility tree:

![Screenshot 2024-11-29 at 8 46 37 AM](https://github.com/user-attachments/assets/063d6365-505e-4d2c-bedd-a6bd28312dbe)

Reply:
I added div's for `display: block`. Any alternative suggestions please?

Reply:
Unfortunately it won't work:

<img width="796" alt="image" src="https://github.com/user-attachments/assets/0b4bb2a6-0098-4745-b0ea-dfa4686ef93a">

Reply:
I would prefer to leave as is and apply display: block to a container rather then a button itself

Reply:
Recap: the CSS for `filter-submit-btn `(our custom class) has width set to 100%. So I would use `vads-u-width--auto` and `vads-u-display--block` to get this instead of using the DIVs and using a custom class for width 100% unnecessarily. Also waiting on answer about which is the right breakpoint for this button because it seems to be turning into 100% width too early.

Approving to fix this and if changes are needed it will be done in a separate ticket as to next mess with staging review.